src/memory_region.rs

```diff
@@ -47,10 +49,31 @@ pub struct MemoryRegion {
     pub vm_gap_shift: u8,
     /// Is also writable (otherwise it is readonly)
     pub is_writable: bool,
+    /// Callback executed when a CoW region is first written to.
+    pub cow_cb: Cell<Option<CowCallback>>,
 }
 
+impl PartialEq for MemoryRegion {
```

Comment:
What do we need this trait implementation for?

Reply:
It was already there, I just made it an explit impl to ignore the cow callback. I think it's only used in tests (where we should probably just compare vmaddrs)